### DISCUSSION TITLE
Return new records from createMany?

### DISCUSSION THREAD

**Discussion by unlocomqx** [prisma/prisma#8348] [Q&A] [answered]
Hi,

Can I make `createMany` return the newly created records instead of the count?
I checked the docs but maybe I'm missing something obvious

I'm using a transaction currently to do the same thing

Thank you

**Comment by ryands17** [comment, marked as answer]
@unlocomqx 👋 

Currently not supported, but we have a related request so it would be great if you could add a 👍 to it: https://github.com/prisma/prisma/issues/8131

**Reply by unlocomqx** [reply]
Thank you very much 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
